gnd/src/main/java/com/google/android/gnd/persistence/local/room/AreaEntity.java

```diff
@@ -20,22 +20,23 @@
 import androidx.room.Entity;
 import androidx.room.PrimaryKey;
 
-import com.google.android.gms.maps.model.LatLng;
 import com.google.android.gms.maps.model.LatLngBounds;
 import com.google.android.gnd.model.basemap.Area;
 import com.google.auto.value.AutoValue;
 
-/**
- * Represents a {@link Area} in the local data store.
- */
+import static com.google.android.gnd.persistence.local.room.LatLngBoundsConverter.boundsFromString;
+import static com.google.android.gnd.persistence.local.room.LatLngBoundsConverter.boundsToString;
+
+/** Represents a {@link Area} in the local data store. */
 @AutoValue
 @Entity(tableName = "area")
 public abstract class AreaEntity {
   public static Area toArea(AreaEntity areaEntity) {
     Area.Builder area =
         Area.newBuilder()
             .setBounds(boundsFromString(areaEntity.getId()))
-            .setState(toAreaState(areaEntity.getState()));
+            .setState(toAreaState(areaEntity.getState()))
+            .setId(areaEntity.getId());
```

Comment:
Nit: Doesn't really matter, but I'd expect setId() to be called first since it's the PK.